### DISCUSSION TITLE
Translations for USACO Guide Content

### DISCUSSION THREAD

**Discussion by maggie-j-liu** [cpinitiative/usaco-guide#2788] [General]
Translating the Guide content was suggested in #2775, but before implementing it, we'd like to know how many people would use this feature and how many people would be willing to help maintain translations.

React with 👍 if you would regularly view Guide content in a language that is not English, or 👎 if this feature wouldn't be useful for you. If it would be useful, comment down below with what language you would use, so we can see how much interest there is in each language.

If you would be willing to help us maintain translations, also leave a comment with the language(s), as well as how much time you think you could spend (joining the team to help consistently, or helping create or review translations once in a while).

Thank you all! :)

**Comment by AhmadFudl** [comment]
Arabic
would give couple hours a week for maintaining the translations

**Reply by MEgooneh** [reply]
Arabic/Persian are RTL so problem isn't just translation.

**Comment by Qubik65536** [comment]
Chinese
I can probably give a few hours for  maintaining the translations

**Reply by Anafaham** [reply]
Chinese i luv u bruh

**Reply by johnzhou721** [reply]
Chinese as well. 2-3h/week

**Comment by orendon** [comment]
Spanish, prolly 1-2 h/week
